server/config-loader.ts

```diff
@@ -24,6 +24,15 @@ export function load(invalidate = false) {
 	const config = ({...defaults, ...require('../config/config')}) as ConfigType;
 	// config.routes is nested - we need to ensure values are set for its keys as well.
 	config.routes = {...defaults.routes, ...config.routes};
+
+	// Automatically disable SQLite modlogs if better-sqlite3 isn't installed
+	try {
+		require('better-sqlite3');
+	} catch (e) {
+		reportError(`Modlogs have been disabled because better-sqlite3 is not installed or could not be loaded.`);
+		config.usesqlitemodlog = false;
```

Comment:
If `usesqlite` is set to `true` in config this should abort startup, only if unset continue operating in degraded mode